epicure-frontend/src/components/CardSwipper/CardSwipper.scss

```diff
@@ -64,7 +60,7 @@
         @include desktop {
             margin-top: 40px;
             justify-content: flex-end;
-            padding-right: 50px;
+            padding-right: 15%;
```

Comment:
Be careful when using % for padding, this might affect responsiveness in a negative way.